lib/pure/typetraits.nim

```diff
@@ -16,12 +16,16 @@ import std/private/since
 export system.`$` # for backward compatibility
 
 type SomeEnumWithHoles* = (not Ordinal) and enum ## Enum with holes.
+type OrdinalEnum* = Ordinal and enum ## Enum without holes.
```

Comment:
Wouldn't it make more sense to call it `SomeOrdinalEnum`, for consistency with `SomeEnumWithHoles` (or rename `SomeEnumWithHoles` to `EnumWithHoles`)?

Reply:
done by renaming `SomeEnumWithHoles` to `HoleyEnum`